components/layout/Header.tsx

```diff
@@ -1,191 +1,212 @@
-                href={item.href}
-                aria-label={`${item.text.toLowerCase()}`}
-                variant="text"
-                // color="neutral"
-                size="small"
-                className="nav-list__item"
+        <Box sx={{ width: '100%' }}>
+          {matches && (
+            <IconButton aria-label="open sidebar menu" onClick={handleDrawerToggle} sx={{ display: 'flex', width: '100%' }}>
+              <Menu
                 sx={{
-                  whiteSpace: 'nowrap', marginLeft: { md: 0 }, px: { md: 2, lg: 3 }, py: 1,
+                  color: theme.palette.text.light, marginLeft: 'auto',
                 }}
-              >
-                {item.text}
-              </Button>
-            ))}
+                fontSize="large"
+              />
+            </IconButton>
+          )}
+          {!matches && (
+            <Box
+              className="desktop-nav-list"
+              sx={{
+                display: 'flex',
+                justifyContent: 'flex-start',
+                width: '450px',
+                paddingRight: '32px',
+              }}
+            >
+              {navItems.map((item, index) => (
+                <Box
+                  key={item.text}
+                  onMouseEnter={() => handleMouseEnter(index)}
+                  onMouseLeave={handleMouseLeave}
+                >
+                  <Button
+                    href={item.href}
+                    aria-label={`${item.text.toLowerCase()}`}
+                    size="small"
+                    className="nav-list__item"
+                    sx={{
+                      whiteSpace: 'nowrap',
+                      marginLeft: { md: 0 },
+                      px: { md: 2, lg: 3 },
+                      py: 1,
+                      '&:hover': {
+                        color: theme.palette.text.secondary,
+                        backgroundColor: theme.palette.primary.main,
+                      },
+                      '&:active': {
+                        color: theme.palette.text.light,
+                        backgroundColor: '#002138',
+                      },
+                    }}
+                  >
+                    {item.text}
+                  </Button>
+                  {hoveredIndex === index && (
+                    <Box
+                      className="dropdown-content"
+                      sx={{
+                        display: 'block',
+                        position: 'absolute',
+                        backgroundColor: theme.palette.primary.dark,
+                        maxWidth: '284px',
+                        boxShadow: '0px 8px 16px 0px rgba(0,0,0,0.2)',
+                        zIndex: 1,
+                      }}
+                    >
+                      {item.sublist?.map((link) => (
+                        <Link key={link} href="/" passHref style={{ textDecoration: 'none' }}>
+                          <Box
+                            component="a"
```

Comment:
The "a" here is denoting that the component will be an anchor for a link.

Reply:
@aitzeng @e-papanicolas 
I am removing `component="a"` because it is causing an unwanted anchor tag nesting issue. The Link component from Next.js already renders an anchor tag, so nesting another anchor tag inside it is invalid and results in a validation warning. Removing `component="a"` will resolve this issue.
![image](https://github.com/user-attachments/assets/058fe2b6-6d20-46c4-8ae0-1dd7b3092561)